react/formspree/src/App.jsx

```diff
@@ -0,0 +1,270 @@
+                  </label>
+                  <input
+                    className="w-full h-10 px-4 rounded-lg outline outline-1 outline-gray-400 hover:outline-gray-600 focus:outline-2 focus:outline-blue-500 transition-all duration-200"
+                    id="previous-role"
+                    name="previous-role"
+                  />
+                </div>
+              </div>
+
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+                <div>
+                  <label className="block text-sm font-medium text-gray-700 mb-1" htmlFor="latest-role-start-date">
+                    Start Date
+                  </label>
+                  <input
+                    className="w-full h-10 px-4 rounded-lg outline outline-1 outline-gray-400 hover:outline-gray-600 focus:outline-2 focus:outline-blue-500 transition-all duration-200"
+                    id="latest-role-start-date"
+                    name="latest-role-start-date"
+                    placeholder="MM-YYYY"
+                  />
+                </div>
+
+                <div>
+                  <label className="block text-sm font-medium text-gray-700 mb-1" htmlFor="latest-role-end-date">
+                    End Date
+                  </label>
+                  <input
+                    className="w-full h-10 px-4 rounded-lg outline outline-1 outline-gray-400 hover:outline-gray-600 focus:outline-2 focus:outline-blue-500 transition-all duration-200"
+                    id="latest-role-end-date"
+                    name="latest-role-end-date"
+                    placeholder="MM-YYYY (Leave blank if current)"
+                  />
+                </div>
+              </div>
+
+              <div>
+                <label className="block text-sm font-medium text-gray-700 mb-1" htmlFor="role-description">
+                  Role Description
+                </label>
+                <textarea
+                  className="w-full px-4 py-2 rounded-lg outline outline-1 outline-gray-400 hover:outline-gray-600 focus:outline-2 focus:outline-blue-500 transition-all duration-200 min-h-[100px]"
+                  id="role-description"
+                  name="role-description"
+                />
+              </div>
+            </div>
+          </div>
+
+          {/* Application Details Section */}
+          <div className="mt-12 space-y-6">
+            <h2 className="text-xl font-semibold text-gray-800 border-b pb-2">Application Details</h2>
+            
+            <div className="space-y-4">
+              <div>
+                <label className="block text-sm font-medium text-gray-700 mb-1" htmlFor="dates">
+                  Position
+                </label>
+                <select
+                  className="w-full h-10 px-4 rounded-lg outline outline-1 outline-gray-400 hover:outline-gray-600 focus:outline-2 focus:outline-blue-500 transition-all duration-200 appearance-none bg-white"
+                  id="dates"
```

Comment:
@Nishantjain10 If this `id` and `name` is meant for `position`, please update the attributes accordingly.